.github/workflows/release-draft.yaml

```diff
@@ -27,7 +27,37 @@ jobs:
       - name: Upload Artifacts
         uses: actions/upload-artifact@v2
         with:
-          name: sysdig-release-${{ env.BUILD_VERSION }}
+          name: sysdig-release-${{ env.BUILD_VERSION }}-x86_64
+          path: /build/release/sysdig-${{ env.BUILD_VERSION }}*
+    
+  build-release-linux-arm64:
+    runs-on: ubuntu-latest
+    env:
+      BUILD_VERSION: ${{ github.ref_name }}
+    steps:
+      - name: Checkout Sysdig
+        uses: actions/checkout@v3
+        with:
+          path: sysdig
+      - name: Create build dir
+        run: |
+          mkdir -p /build
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: 'arm64'
+      - name: Run the build process with Docker
+        uses: addnab/docker-run-action@v3
+        with:
+          image: draios/sysdig-builder:dev
+          registry: ghcr.io
+          options: --platform=linux/arm64 -v ${{ github.workspace }}/sysdig:/source/sysdig -v /build:/build
```

Comment:
```
-v /build:/build
```
Hopefully this will be enough to persist in the workspace the artifacts created through `docker-run-action`. I think our only option is to test this with an rc tag and see how it goes.